sdk/ai/Azure.AI.Projects/samples/Sample7_AzureOpenAI.md

```diff
@@ -14,8 +14,8 @@ This sample demonstrates how to use the synchronous and asynchronous `Azure Open
 ```C# Snippet:AzureOpenAISync
 var endpoint = System.Environment.GetEnvironmentVariable("PROJECT_ENDPOINT");
 var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
-AIProjectClient projectClient = new(new Uri(endpoint), new DefaultAzureCredential());
-ChatClient chatClient = projectClient.GetAzureOpenAIChatClient(modelDeploymentName);
+AIProjectClient projectClient = new AIProjectClient(new Uri(endpoint), new DefaultAzureCredential());
+ChatClient chatClient = projectClient.GetAzureOpenAIChatClient(connectionName: null, apiVersion: null, deploymentName: modelDeploymentName);
```

Comment:
NIT: Should deploymentName be the first parameter, since it's probably more likely to be required?